ev2/event-driven/vis/IPT.cpp

```diff
@@ -164,6 +177,20 @@ const cv::Mat& vIPT::getQ(){
     return Q;
 }
 
+const cv::Mat& vIPT::getK(int cam) {
```

Comment:
what happens if cam = 2. we would return cam[0]? it could introduce error.
what about
if( cam == 0 || cam == 1 ) return cam_matrix[cam]
else return empty matrix?